lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -5,84 +5,119 @@
+
+fdescribe('NxPageHeader', function() {
+
+  const renderEl = rtlRenderElement(NxPageHeader, {});
 
-  it('renders an AbstractNxPageHeader', function() {
-    expect(getShallowComponent()).toMatchSelector(AbstractNxPageHeader);
+ it('renders a simple NxPageHeader', function() {
+    renderEl();
+    const logo = screen.getByRole('img')
+    expect(logo).toBeInTheDocument();
+    
   });
 
-  it('passes its links, homeLink, and children props to the AbstractNxPageHeader', function() {
-    const props = {
-          links: [{ name: 'foo', href: '#bar' }],
-          homeLink: '#home',
-          children: <button>Click Here For a Free iPhone</button>
-        },
-        component = getShallowComponent(props);
+  it('renders a custom logo when given logo prop', function() {
+    const logo = renderEl({ logo: { path: 'foo', alt: 'bar' } })?.querySelector("img");
+
+    expect(logo).toHaveAttribute('src', 'foo');
+    expect(renderEl({ logo: { path: 'foo', alt: 'bar' } })?.querySelectorAll("img").length).toEqual(1);
+  });
+
+  it('renders logo link when given homeLink prop', function() {
+    renderEl({ homeLink: '#home'});    
 
-    expect(component).toHaveProp('links', props.links);
-    expect(component).toHaveProp('homeLink', props.homeLink);
-    expect(component.children()).toMatchElement(props.children);
+    const homeLink = screen.getByRole('link', {name: 'Home'}),
+      homeLinkLogo = homeLink.querySelector('img');
+
+    expect(homeLink).toBeInTheDocument();
+    expect(homeLinkLogo).toBeInTheDocument();
   });
 
-  it('passes a logo with a webpack-generated path and proper alt tag', function() {
-    const logo = shallow(getShallowComponent().prop('logo'));
+  it('renders links in header when given links prop', function() {
+    renderEl({links: [{ name: 'foo', href: '#bar' }, { name: 'baz', href: '#qux' }]});
+
+    const link1 = screen.getByRole('link', {name: 'foo'}),
+      link2 = screen.getByRole('link', {name: 'baz'})
 
-    // this path comes from imgMock.ts
-    expect(logo).toHaveDisplayName('img');
-    expect(logo).toHaveProp('src', 'path/to/asset.png');
-    expect(logo).toHaveProp('alt', 'Sonatype');
-    expect(logo).toHaveClassName('nx-product__logo-image');
+    expect(link1).toBeInTheDocument();
+    expect(link2).toBeInTheDocument();
   });
 
-  it('includes the productInfo.name in the productInfoContent .nx-product__name if provided', function() {
+  it('renders the product name if provided', function() {
     const props = {
           productInfo: {
             name: 'test app'
           }
         },
-        defaultProductInfoContent = getShallowComponent().prop('productInfoContent'),
-        richProductInfoContent = shallow(getShallowComponent(props).prop('productInfoContent'));
+        header = renderEl(),
+        headerWithProductName = renderEl(props),
+        headerWithProductNameText = headerWithProductName?.querySelector('div.nx-product__name');
```

Comment:
The element tagname and classname are implementation details.  Just assert that the header overall contains the `name` text.

Reply:
> The element tagname and classname are implementation details. Just assert that the header overall contains the `name` text.

Done: [15b9325](https://github.com/sonatype/sonatype-react-shared-components/pull/935/commits/15b9325c914cc126a4aa0df665457670492f84c9)